[2021-07-30 14:03:42,649] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:48:22.193748+00:00 [queued]>
[2021-07-30 14:03:42,673] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:48:22.193748+00:00 [queued]>
[2021-07-30 14:03:42,674] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-30 14:03:42,675] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-30 14:03:42,676] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-30 14:03:42,700] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-22T13:48:22.193748+00:00
[2021-07-30 14:03:42,713] {standard_task_runner.py:52} INFO - Started process 717 to run task
[2021-07-30 14:03:42,720] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-22T13:48:22.193748+00:00', '--job-id', '2083', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmp4ypswag3', '--error-file', '/tmp/tmpm5mhjnts']
[2021-07-30 14:03:42,721] {standard_task_runner.py:77} INFO - Job 2083: Subtask restart_test
[2021-07-30 14:03:42,843] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-22T13:48:22.193748+00:00 [running]> on host 04be2259e97a
[2021-07-30 14:03:42,925] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-22T13:48:22.193748+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-22T13:48:17.845901+00:00
[2021-07-30 14:03:49,229] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210722T134822, start_date=20210730T140342, end_date=20210730T140349
[2021-07-30 14:03:49,272] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-30 14:03:49,310] {local_task_job.py:151} INFO - Task exited with return code 0
